pkg/apis/vpc/v1alpha1/ipaddressallocation_types.go

```diff
@@ -41,20 +40,20 @@ type IPAddressAllocationList struct {
 
 // IPAddressAllocationSpec defines the desired state of IPAddressAllocation.
 type IPAddressAllocationSpec struct {
-	// IPAddressBlockVisibility specifies the visibility of the IPBlocks to allocate IP addresses. Can be External or Private.
-	// +kubebuilder:validation:Enum=External;Private
+	// IPAddressBlockVisibility specifies the visibility of the IPBlocks to allocate IP addresses. Can be External, Private or PrivateTGW.
+	// +kubebuilder:validation:Enum=External;Private;PrivateTGW
 	// +kubebuilder:default=Private
 	// +optional
-	IPAddressBlockVisibility IPAddressVisibility `json:"ip_address_block_visibility,omitempty"`
-	// AllocationSize specifies the size of IP CIDR to be allocated.
-	AllocationSize int `json:"allocation_size,omitempty"`
+	IPAddressBlockVisibility IPAddressVisibility `json:"ipAddressBlockVisibility,omitempty"`
+	// AllocationSize specifies the size of allocation_ips to be allocated.
+	AllocationSize int `json:"allocationSize,omitempty"`
 }
 
 // IPAddressAllocationStatus defines the observed state of IPAddressAllocation.
 type IPAddressAllocationStatus struct {
-	// CIDR is the allocated CIDR
-	CIDR       string      `json:"CIDR"`
-	Conditions []Condition `json:"conditions,omitempty"`
+	// allocation_ips is the allocated IP addresses
```

Comment:
`allocation_ips` -> `AllocationIPs`, keep the same with L48 (`AllocationSize specifies ...`)